### PR TITLE
[Feat] [Translations] [Spanish]: advanced/reliability

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]

File: site/docs/es/advanced/reliability.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+```js
+const { Bot } = require("grammy");
+const { run } = require("@grammyjs/runner");
+
+const bot = new Bot("<token>");
+
+const runner = run(bot);
+
+// Stopping the bot when Node process
+// is about to be terminated
```
Comment (roj1512) [inline site/docs/es/advanced/reliability.md:109]: Missed

File: site/docs/es/advanced/reliability.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+```ts
+import { Bot } from "https://deno.land/x/grammy/mod.ts";
+import { run } from "https://deno.land/x/grammy_runner/mod.ts";
+
+const bot = new Bot("<token>");
+
+const runner = run(bot);
+
+// Stopping the bot when Node process
+// is about to be terminated
```
Comment (roj1512) [inline site/docs/es/advanced/reliability.md:127]: Missed

File: site/docs/es/advanced/reliability.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+```ts
+import { Bot } from "grammy";
+import { run } from "@grammyjs/runner";
+
+const bot = new Bot("<token>");
+
+const runner = run(bot);
+
+// Stopping the bot when Node process
+// is about to be terminated
```
Comment (roj1512) [inline site/docs/es/advanced/reliability.md:90]: Missed

**Comment by habemuscode** [comment]
@roj1512 Done

**Pull request merged**